_data/fall_2016/roycboyc.json

```diff
@@ -1,4 +1,4 @@
 {
-  "emoji": "brifcase"
+  "emoji": ":brifcase:",
```

Comment:
"brifcase" is misspelled. Also you don't need ": :"
E.g. https://github.com/startup-systems/students/pull/44/files